### PR TITLE
feat(3000): Add plus button back to product analytics

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

Adds the plus:

Screenshot 2023-12-06 at 12 35 14

### CODE REVIEW

File: frontend/src/layout/navigation-3000/navigationLogic.tsx [hunk 1]
```diff
@@ -364,6 +365,12 @@ export const navigation3000Logic = kea<navigation3000LogicType>([
                             icon: <IconGraph />,
                             logic: isUsingSidebar ? insightsSidebarLogic : undefined,
                             to: isUsingSidebar ? undefined : urls.savedInsights(),
+                            sideAction: {
+                                icon: <IconPlusMini />, // The regular plus is too big
```
Comment (Twixes) [inline frontend/src/layout/navigation-3000/navigationLogic.tsx:369]: @corywatilo `IconPlus` in @posthog/icons is really big! If that's Central's rendition of a plus, then I think it'd be practical to make it just a bit smaller. WDYT?

File: frontend/__snapshots__/scenes-other-onboarding--onboarding-other-products.png
Comment (Twixes) [inline frontend/__snapshots__/scenes-other-onboarding--onboarding-other-products.png:1]: 🤔

File: frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss [hunk 2]
```diff
@@ -80,14 +80,14 @@
             --lemon-button-font-size: 0.75rem;
             --lemon-button-height: 1.6875rem;
             --lemon-button-gap: 0.25rem;
-            --lemon-button-side-action-icon-width: 1.25rem;
+            --lemon-button-side-action-width: 1.5rem;
```
Comment (daibhin) [inline frontend/src/lib/lemon-ui/LemonButton/LemonButton3000.scss:83]: Should this value be computed or configurable?

In the case of the small & default options it seems the new `--lemon-button-side-action-width` equals the old `--lemon-button-side-action-icon-width` + `--lemon-button-padding-horizontal`. 

In this case there is a drift (`0.375rem + 1.25rem =/= 1.5rem`). I'm surprised no snapshots changed but my guess is that we don't snapshot the `xsmall` button with a side action

**Review by daibhin — APPROVED** [review]
Going to approve this to unblock 3000 and we can come back to the sizing of side icons on xsmall buttons in a follow-up

**Pull request merged**